app/src/test/java/net/squanchy/search/engines/EventSearchEngineTest.java

```diff
@@ -1,124 +0,0 @@
-package net.squanchy.search.engines;
-
-import net.squanchy.schedule.domain.view.Event;
-
-import org.junit.Test;
-
-import static net.squanchy.schedule.domain.view.EventFixtures.anEvent;
-import static org.fest.assertions.api.Assertions.assertThat;
-
-public class EventSearchEngineTest {
```

Comment:
Converted to Kotlin (couldn't wait here and in another place because of the changes to the fixtures anyway invalidating them)